docs-markdown/src/controllers/xref-controller.ts

```diff
@@ -164,10 +165,18 @@ async function getXrefDisplayProperty() {
 
 async function getXrefSelection() {
   const items: QuickPickItem[] = [];
-  let uid: string | undefined = await window.showInputBox({ placeHolder: "Enter XREF Search Term" });
+  const uid: string | undefined = await window.showInputBox({ placeHolder: "Enter XREF Search Term" });
   if (uid) {
-    let content = await getAsync(`${rootUrl}/autocomplete?tags=${tags}&text=${uid}`);
-    content.data.map((item: { tags: any; uid: string; }) => {
+    const response = await getAsync(`${rootUrl}/autocomplete?tags=${tags}&text=${uid}`);
+    if (response.status !== 200) {
```

Comment:
Main change is to add Error message response if status code is not 200, and if there are no results returned on line 175